docs/Modders/Unique-parameters.md

```diff
@@ -206,6 +206,11 @@ For example: `+1 Science`.
 
 These can be strung together with ", " between them, for example: `+2 Production, +3 Food`.
 
+## resourceFilter
+
+At the moment, only used for the `"Improves [resourceFilter] resource in this tile"` Unique on Improvements.
```

Comment:
Please turn this into "allowed values" list, like most of them are

Reply:
A markdown list, with bullet points, like most of the parameters are displayed in this page